canonical-websites/www.ubuntu.com
# Remove 1710 page and redirect to desktop features

## Done

Remove 1710 page and redirect to `/desktop/features`

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/desktop/1710>
- Ensure that the page is redirected to `/desktop/features`


## Issue / Card
Fixes #2910
